tests/unit/components/file-uploader/FileUploader.spec.js

```diff
@@ -0,0 +1,52 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+// import userEvent from '@testing-library/user-event'
+import FileUploader from '~/components/file-uploader/FileUploader'
+
+const buttonText = 'test'
+const fakeFile = new File(['certificate'], 'test-file.png', { type: 'application/pdf' })
+describe('FileUploader test', () => {
+  beforeEach(() => {
+    render(<FileUploader buttonText={ buttonText } />)
+  })
+
+  it('should render button with upload icon', () => {
+    const button = screen.getByText('test')
+    const icon = screen.getByTestId('CloudUploadIcon')
+
+    expect(icon).toBeInTheDocument()
+    expect(button).toBeInTheDocument()
+  })
+    
+  // it('should upload file', async () => {
```

Comment:
Add a description with @TODO, or remove the comment.

Reply:
fixed